doc/ocp_lock/lock_spec.ocp

```diff
@@ -1507,6 +1493,93 @@ Table: Next action values {#tbl:next-action-values}
+
+Table: SealedOldAndNewAccessKey contents
+
++-----------------+-----------------------+----------------------------------------------+
+| Name            | Type                  | Description                                  |
++=================+=======================+==============================================+
+| old_access_key  | SealedAccessKey       | Old access key. Used to decrypt the new      |
+|                 |                       | access key.                                  |
++-----------------+-----------------------+----------------------------------------------+
+| ak_iv           | u8[Nn]                | IV used to decrypt the encrypted access key. |
++-----------------+-----------------------+----------------------------------------------+
+| encrypted_ak_ct | u8[access_key_len+Nt] | (Encrypted access key + tag) ciphertext.     |
++-----------------+-----------------------+----------------------------------------------+
+| tag             | u8[Nt]                | Authentication tag for AES operation         |
++-----------------+-----------------------+----------------------------------------------+
+
+`Nn` and `Nt` are HPKE values associated with the `aead_id` identifier from the given `hpke_algorithm`.
+
+The outer layer of encryption of the wrapped access key is decrypted using HPKE. The inner layer is decrypted using the access key in the accompanying `SealedAccessKey` instance. Both layers use the AEAD algorithm indicated by the `aead_id` identifier from the given `hpke_algorithm`.
+
+##### WrappedKey type
+
+AES-256-GCM is used for all wrapping and unwrapping.
+
+Table: WrappedKey contents
+
++----------+------------+--------------------------------------------+
+| Name     | Type       | Description                                |
++==========+============+============================================+
+| key_type | u16        | Type of the wrapped key. This is treated   |
+|          |            | as AAD when the key is decrypted.          |
+|          |            |                                            |
+|          |            | - 0h: Reserved                             |
+|          |            | - 1h: LOCKED_MPK (ciphertext held at rest) |
+|          |            | - 2h: READY_MPK (ciphertext held in RAM)   |
+|          |            | - 3h: WRAPPED_MEK                          |
+|          |            | - 4h to FFFFh: Reserved                    |
++----------+------------+--------------------------------------------+
+| iv       | u8[12]     | Initialization vector for AES operation.   |
++----------+------------+--------------------------------------------+
+| ct_len   | u32        | Length of the ciphertext in bytes.         |
```

Comment:
Could probably reduce this down to `u16` and move next to `key_type` for alignment purposes.

Reply:
That's a good suggestion, I'll add that in a follow on PR.